docs/source/openvino/openvino.rst

```diff
@@ -0,0 +1,72 @@
+Enabling OpenVINO™
+~~~~~~~~~~~~~~~~~~
+
+This tutorial introduces how to enable OpenVINO™ for GQCNN deployment on Intel® devices.
+
+Intel® Distribution of OpenVINO™ (Open Visual Inference & Neural network Optimization) toolkit, based on convolutional neural networks (CNNs), extends computer visoin workloads across Intel® hardware (including accelerators) and maximizes performance. The toolkit enables deep learning inference at the edge computation, and supports heterogeneous execution across various compution vision devices -- CPU, GPU, Intel® Movidius™ NCS2, and FPGA -- using a **common** API.
+
+.. image:: ../images/gqcnn_openvino.png
+   :width: 440 px
+   :height: 250 px
+   :align: center
+
+Install OpenVINO™ Toolkit
+=========================
+The toolkit is available from open source project `Intel® OpenVINO™ Toolkit`_.
+
+.. note:: GQCNN uses two layers, RandomUniform and Floor, which are not supported by the 2019_R3 release of OpenVINO™. `PR #338 <https://github.com/opencv/dldt/pull/338>`_ adds the support for these two layers.
```

Comment:
I didn't have these issues here: https://github.com/BerkeleyAutomation/gqcnn/blob/ncs/gqcnn/model/tf/ncs_fc_network_tf.py#L242, so I'm curious as to what the difference is. One is that I use `self._graph.as_graph_def()` instead of `self._sess.graph_def`, however I don't think that's it. Another is that the method in which the protobufs are written is a bit different. Finally I guess you are using OpenVINO whereas I was using the NCSDK, but I would think that they would support similar operations, right?

I also think it's a bit weird that the frozen graph would require RandomUniform because that should be strictly for training. If I recall correctly, only operations that the output directly depends on (i.e. things for inference) should be kept by: https://github.com/BerkeleyAutomation/gqcnn/blob/ncs/gqcnn/model/tf/ncs_fc_network_tf.py#L245. I don't exactly remember when/where Floor is used. 

This might be worth looking into until your PR for the RandomUniform and Floor operations is approved. Even if you could just copy my code and try it out, that would be super helpful. Maybe something changed and it will still break.

Reply:
A little bit surprised hearing from you that RandomUniform or Floor is not used... 

The OpenVINO Model Optimizer tool detects the exist of these two layers. Besides, the [tensorboard](https://github.com/tensorflow/tensorboard) visualization tool also illustrates these two layers, for pre-trained GQCNN 2.0, 2.1, 3.0, 4.0-pj, 4.0-suc downloaded by [download_models.sh](https://github.com/BerkeleyAutomation/gqcnn/blob/master/scripts/downloads/models/download_models.sh). I'm attaching the screen snapshot from tensorboard client, which might help you to recall something... :)

The usage of RandomUniform or Floor should not be introduced by OpenVINO or NCSDK. Both tools are inference stacks. While these two layers are observed in the original GQCNN graphs.

I also noticed these two layers were not introduced in the pre-trained models at Aug. 2018, when I started some works on GQCNN acceleration. My initial guessing is they were added later.

I won't think it related to `self._graph.as_graph_def()` or `self._sess.graph_def`, these two are equal if check `==` while freezing.